lib/ansible/modules/network/ios/ios_vlan.py

```diff
@@ -129,53 +129,53 @@ def map_obj_to_commands(updates, module):
 
         if state == 'absent':
             if obj_in_have:
-                commands.append('no vlan {}'.format(vlan_id))
+                commands.append('no vlan {0}'.format(vlan_id))
```

Comment:
Why did you do this unrelated change? Do you want to support Python 2.6?

Reply:
What do you mean by checking box?

Reply:
I don't know a lot about testing but by default the python 2.6 compatibility is checked by shippable but i don't know if it's needed. By checking box I mean to have all checks of shippable successful.